fundingmanager.go

```diff
@@ -120,8 +120,25 @@ func (r *reservationWithCtx) updateTimestamp() {
 // embedded within this message giving the funding manager full context w.r.t
 // the workflow.
 type InitFundingMsg struct {
-	peer lnpeer.Peer
-	*openChanReq
+	Peer             lnpeer.Peer
+	TargetPubkey     *btcec.PublicKey
+	ChainHash        chainhash.Hash
```

Comment:
nit: could add some godocs too.

Reply:
This will replace `openChanReq` so going to refactor a little and add godocs.